Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -134,10 +133,11 @@ internal final class DiscoveryPageViewController: UITableViewController {
       |> \.rowHeight .~ UITableView.automaticDimension
       |> \.estimatedRowHeight .~ 200.0
 
-    if let preferredBackgroundColor = self.preferredBackgroundColor {
-      _ = self
-        |> \.view.backgroundColor .~ preferredBackgroundColor
-    }
+    _ = self.view
+      |> \.backgroundColor .~ (
+        // Update the background if it is not currently clear (contained in EditorialProjectsViewController)
+        self.view.backgroundColor != .clear ? discoveryPageBackgroundColor() : self.view.backgroundColor
```

Comment:
Unfortunately snapshot tests wouldn't pass without this - it seems setting the `backgroundColor` any earlier than `bindStyles` was ineffective. We only ever set the `backgroundColor` to `.clear` when this is contained in `EditorialProjectsViewController`.